### PR TITLE
routes

### PR DESCRIPTION
**Pull request by Wissam-nsr**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,18 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "pages#home"
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get "uikit", to: static("front.html.erb")
+
+  resources "users", only [:show] do
```
Comment (ddlire) [inline config/routes.rb:6]: /!\ :users et non "users"
as: profile ?

File: config/routes.rb [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+    resources "trips", only [:new, :create, :update] do
```
Comment (ddlire) [inline config/routes.rb:7]: :trips 
adding destroy ?

File: config/routes.rb [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
+      resources "moments", only [:index, :create]
```
Comment (ddlire) [inline config/routes.rb:8]: :moments
no index but :new needed
missing the steps nested as moments
options (no need to nest):  edit, update and destroy

File: config/routes.rb [hunk 4]
```diff
@@ -1,8 +1,18 @@
...
+    end
+  end
+
+  resources "chatrooms", only [:index, :show, :create, :destroy] do
+    resources "messages", only: [:create]
+  end
+
+  resources "steps", only [:index, :new, :create, :destroy]
```
Comment (ddlire) [inline config/routes.rb:16]: new & create nested in trip 
no index
destroy ok

File: config/routes.rb (hunk 2 above)
Comment (Wissam-nsr) [reply config/routes.rb:7]: ok

File: config/routes.rb (hunk 3 above)
Comment (Wissam-nsr) [reply config/routes.rb:8]: i think we need index for the profile page

**Pull request merged**